source/components/cardContainer/filters/filterGroup/filterGroup.directive.ng1.ts

```diff
@@ -54,5 +68,6 @@ export let filterGroup: angular.IComponentOptions = {
 		icon: '=',
 		filterGroup: '=',
 		source: '=',
+		enabled: '<',
```

Comment:
So after some discussion with @SFarageNIS, we recognized that the cases when you'd say `!disabled` or `disabled == false` are actually pretty rare. By default the component will always be enabled. So you'd only ever specify `disabled` if you specifically wanted to disable it under certain conditions. So at this point we'd like to just stick with using disabled, since it would take some migration to get `enabled` as a consistent pattern.